src/views/payment-sheet-views/card-view.js

```diff
@@ -73,7 +72,7 @@ CardView.prototype.initialize = function () {
   }
 
   if (!this.model.isGuestCheckout && this.merchantConfiguration.vault.allowVaultCardOverride === true) {
-    classList.remove(this.getElementById('save-card-field-group'), 'braintree-hidden');
+    this.getElementById('save-card-field-group').classList.remove('braintree-hidden');
```

Comment:
Is this `getElementById` guaranteed to not return `null`?

Reply:
Checked `@braintree/class-list` and it looks like it doesn't do a check there, so this is probably okay.